fake-koji/src/main/java/org/fakekoji/http/ProjectMapping.java

```diff
@@ -104,7 +104,7 @@ String getProductOfProject(String project, List<String> productList, List<String
         throw new ProjectDoesNotMatchException();
     }
 
-    List<String> getExpectedArchesOfProject(String project) throws ProjectMappingException {
+    public List<String> getExpectedArchesOfProject(String project) throws ProjectMappingException {
         File expectedArchesFile = null;
```

Comment:
All methods which are wxposed via curl api, should be public. Although it is not related, pelase include